lib/Semantics/mod-file.cpp

```diff
@@ -157,13 +158,17 @@ std::string ModFileWriter::GetAsString(const Symbol &symbol) {
     all << ") " << symbol.name();
   }
   all << '\n' << uses_.str();
-  uses_.str(""s);
+  uses_.flush();
+  uses_Buf.clear();
```

Comment:
Can this be uses_.str().clear() ?

Reply:
I really dont know, because when I grep on llvm  there is only one use of something.str().clear. But lots of something.clear() for llvm::raw_string_stream.
@DavidTruby what do you think?

Reply:
Talking with @DavidTruby it looks either is fine. But maybe the one proposed by Kiran in cleaner.

Reply:
Done!